CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - Fixed permissions checks in the UI that prevented users with certain limited permissions from creating report configurations.
 - ROX-9946: Fixed default permissions for the default Vuln Reporter role to exclude the modify permission on notifiers, since it is not needed for report creation.
 - Added AllowPrivilegeEscalation as a new policy criteria.
+- Introduction of `--image-default=opensource` to `roxctl generate` commands. This will default image registries to the public Open Source images at `quay.io/stackrox-io`.
```

Comment:
[I did not comment on https://srox.slack.com/archives/C02HX541YQN/p1649925622507029 because I wanted to see it first :-P ]
I think announcing this makes sense only when we're done with overall open source image flavor topic and when users are able to actually pull and install images from quay.io/stackrox-io. Let's hold adding changelog item until then?